fabric-common/types/index.d.ts

```diff
@@ -225,9 +224,14 @@ export class DiscoveryService extends ServiceAction {
 	public getDiscoveryResults(refresh?: boolean): Promise<any>;
 	public close(): void;
 }
+export interface RegistrationOpts {
+	startBlock?: number|string|Long;
+	endBlock?: number|string|Long;
+	unregister?: boolean;
+}
 
 export class EventListener {
-	constructor(listenerType: string, callback: any, options: any, event: string);
+	constructor(eventService: EventService, listenerType: string, callback: any, options?: RegistrationOpts, event?: RegExp|string, chaincodeId?: string);
 	onEvent(error: Error, event: any): void;
```

Comment:
This looks to me like it shouldn't be in the public API either

Reply:
I believe a exported interface is a good structure of it